### PR TITLE
Display hex values with monospace font

### LINKED ISSUE #99: Improve User Interface visibility
## Motivation

**I felt that the visibility was relatively low compared to other blockchain explorers.** It seems that we should reach the level of other services that will be visited by ordinary users in the future through the Nine Chronicle Launcher.

Screen_Shot_2020-08-04_at_2 05 42_PM

### Reference service

- [Blockchain Explorer | ETH](https://www.blockchain.com/explorer?view=eth)
- [etherscan.io](https://etherscan.io)

### Main page

Screen_Shot_2020-08-04_at_12 18 55_PM

- [ ]  Increase the width of the **Transaction** column. it is difficult to see if the number of two digits exceeds.
- [ ]  Increase the width of the **index** column. Same reason as below item.
- [ ]  Removed **difficulty** column. It is not helpful to identify a specific block.
- [ ] ~~Show block index or 6-length abstract-hash instead of 64 length hash.~~ Not yet, but it would be useful When it becomes searchable types.

Screen_Shot_2020-08-04_at_12 19 15_PM

- [ ] Adding a UI to show a transaction list. like upper screenshot.

### Blocks and Transactions page

- [ ] Show the block and transaction informations as a table.

### Transaction page

- [ ] Show the action like below code block.

```json
{
  "values": {
    "foods": [],
    "worldId": "1",
    "stageId": "11",
    "weeklyArenaAddress": "cv1HYOuUly92BmrJTeWJteY7W+o=",
    "costumes": [],
    "equipments": [
      "X6PtkJOhrE220ugX36vV4A==",
      "lh8uTHK4XkarIIlD+slRyQ==",
      "RT/E96KgPESUKFcczf+90Q=="
    ],
    "id": "KUk/GU9zDke/jlPtTYf1qw==",
    "avatarAddress": "2Q39HkxyaoJSRceRTzVFTPBIs8Q="
  },
  "type_id": "hack_and_slash"
}
```

### LINKED ISSUE #115: Display hex values with monospace font
It's a visibility improvement suggestion in addition to #99. Now hex values in main pages use same font with other elements. It would be great if we use *monospace font* (i.e. `consolas`) for better visibility.

### PR DESCRIPTION
**Pull request by mrbartrns**
I added monospace font-style to link in Account Details page and codes in Transaction Details page.

account_details
transaction_details

 close #115

### CODE REVIEW

File: src/index.css [hunk 1]
```diff
@@ -32,3 +32,11 @@ h1 {
...
   margin-bottom: 10px;
 }
+
+.ms-List-cell a {
+  font-family: 'Consolas', monospace;
+}
+
+code {
+  font-family: 'Consolas', monospace;
+}
```
Comment (longfin) [inline src/index.css:42]:
```suggestion
}

```

**Review by longfin — COMMENTED** [review]
Thanks for your contribution! I left a comment (small nitpicking 😅 )

**Review by limebell — APPROVED** [review]
🎉

File: src/index.css [hunk 2]
```diff
@@ -32,3 +32,7 @@ h1 {
 .nav {
   margin-bottom: 10px;
 }
+
+.ms-List-cell a{
```
Comment (x86chi) [inline src/index.css]:
```suggestion
.ms-Link {
```

I think it's better to use `.ms-Link` selector

**Review by x86chi — CHANGES_REQUESTED** [review]

File: src/index.css (hunk 2 above)
Comment (mrbartrns) [reply src/index.css]: thanks for your feedback!

**Review by dahlia — APPROVED** [review]

**Review by dahlia — APPROVED** [review]

**Review by x86chi — APPROVED** [review]

**Review by x86chi — CHANGES_REQUESTED** [review]
Could you squash into two commit like below? The version history is too long.
1. Display hex values with monospace font
2. Shows the consolas font first for action block.
  The default monospace font in the windows is less readable.

**Comment by mrbartrns** [comment]
Done! Please check again.

**Review by longfin — APPROVED** [review]

**Review by x86chi — APPROVED** [review]
Thank you for squash commit. In my opinion, it was better to clearly separate the history for each revision.

> 1. Display hex values with monospace font
> 2. Shows the consolas font first for action block.
> 
> As I said in previous comments: https://github.com/planetarium/libplanet-explorer-frontend/pull/116#pullrequestreview-484253988

**Review by limebell — APPROVED** [review]

**Review by dahlia — APPROVED** [review]

**Comment by dahlia** [comment]
Thank you for the first contribution!

**Pull request merged**